torch/onnx/_internal/fx/passes/modularization.py

```diff
@@ -818,6 +828,39 @@ class Modularize(_pass.Transform):
 
     """
 
+    @_beartype.beartype
+    def __init__(
+        self,
+        diagnostic_context: diagnostics.DiagnosticContext,
+        module: torch.fx.GraphModule,
+        exported_program: Optional[torch.export.ExportedProgram] = None,
+    ):
+        super().__init__(diagnostic_context, module)
+        self.module = module
+        self.is_exported_program = exported_program is not None
+        self.exported_program = (
+            exported_program if exported_program is not None else None
+        )
+        self.graph_signature = (
```

Comment:
out of scope of this PR, but I wonder what is the cost to have something similar for non exported_program path?

Reply:
To construct graph_signature? Probably AOT export

Reply:
We could instantiate a graphsignature class and build it ourselves